.github/workflows/ci.yml

```diff
@@ -0,0 +1,127 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+      - '!*backport*'
+    tags:
+      - 'v*'
+      - '!*dev*'
+      - '!*pre*'
+      - '!*post*'
+  pull_request:
+  # Allow manual runs through the web UI
+  workflow_dispatch:
+
+jobs:
+  core:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@main
+    with:
+      submodules: false
+      coverage: codecov
+      toxdeps: tox-pypi-filter
+      posargs: -n auto
+      libraries: |
+        apt:
+          - libopenjp2-7
+      envs: |
+        - linux: py39
+
+  test:
+    needs: [core]
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@main
+    with:
+      submodules: false
+      coverage: codecov
+      toxdeps: tox-pypi-filter
+      posargs: -n auto
+      libraries: |
+        apt:
+          - libopenjp2-7
+        brew:
+          - openjpeg
+      envs: |
+        - macos: py38
+        - windows: py310
+        - linux: py38-oldestdeps
+        - linux: py39-conda
+          libraries: ''
```

Comment:
Why does macOS and Linux (apparently) need a JP2 library while Windows and conda on Linux does not?

Reply:
If I recall, at the time, we couldn't install the JP2 library on windows so it was skipped (along with those tests).
For conda, JP2 is a dependency of the library we use and is installed via conda-forge.

Reply:
If we have a way to install the jp2 lib on windows we should and then we can run the tests.